.github/workflows/upstream-pr.yaml

```
@@ -0,0 +1,88 @@
+name: Create PR in upstream repo
+
+on:
+  push:
+    branches:
+      - 'main'
```

This repo is still using `master` for the default branch
```suggestion
      - 'master'
```

---

Good spot